docs/developer-docs/integrations/sns/managing/making-proposals.md

```diff
@@ -742,6 +789,20 @@ quill send message.json
 ```
 
 
+## Critical proposals
+
+Some proposal types are considered "critical". These are DeregisterDappCanisters, TransferSnsTreasuryFunds, and MintSnsTokens. Critical proposal types are handled differently to ensure they are only passed with broad community consensus.
+
+1. Voting Thresholds
+
+   Non-critical proposals types can be passed if 3% of the total voting power votes yes and 50% of the exercised voting power votes yes.
+
+   Critical proposals types can only be passed if 20% of the total voting power votes yes and 67% of the exercised voting power votes yes. 
+
+2. Catchall following
+
+   Users must set following explicitly for critical proposal types.
```

Comment:
I would add a setence here what catch-all following means. 
Maybe we can just copy some of the explanation from the forum post, e.g., a part of "Remove critical proposals from the catch-all “All topics”: Voters can follow other neurons on critical proposals, but each neuron has to make an active decision of following for each critical proposals type as the “All topics”-following is not applied to critical proposals. Users who have multiple neurons can actively vote with just one of them and follow this one neuron with all their other neurons.
It would be possible to reset following on critical proposals in all existing SNSs as an addition to this measure.
Details: This is an alternative to “Remove following for critical proposals” as proposed above that was proposed by the community. This means that the complete removal of following would not be pursued anymore.
Next steps: If the community does not express concerns here, we plan to remove critical proposals from the catch-all and propose it to the NNS as part of the normal SNS release process. We would then discuss in a separate forum thread whether, in addition, following on critical proposals should also be reset in existing SNSs. This addition can then be done separately if this is what the community agrees on." ?

Reply:
Great point!